README.md

```diff
@@ -55,6 +55,22 @@ function main(responses) {
 }
 ```
 
+### Request and response cycle
+
+The request and response are linked. If you create a request without a corresponding response, by default the request will not be sent. If you want to send the request without listening for a response, you must specify `eager: true` for your request object:
```

Comment:
I am not sure "linked" is the best word to describe it. How about:

Requests behave lazy by default. They will not be sent until their response stream gets subscribed.